apps/projects/app/components/Shared/FilterBar/prepareFilters.js

```diff
@@ -0,0 +1,86 @@
+import { BOUNTY_STATUS } from '../../../utils/bounty-status'
+
+/*
+  prepareFilters builds data structure for displaying filterbar dropdowns
+  data comes from complete issues array, issuesFiltered is used for counters
+*/
+const prepareFilters = (issues, bountyIssues) => {
```

Comment:
FYI no changes were made to this function - it's identical to the one stripped out of the `FilterBar` component